docs/secure-connections.md

```diff
@@ -94,10 +95,11 @@ Sauce IPSec Proxy establishes a VPN connection between two IPSec gateways, and a
   </div>
 </div>
 
+For more information, see our [Sauce IPSec Proxy white paper](https://saucelabs.com/resources/white-papers/sauce-ipsec-proxy-overview), or contact your Customer Success Manager or Sauce Labs Sales Engineer.
 
 ## Choosing the Right Solution
 
-Sauce Connect Proxy and Sauce IPSec Proxy accomplish the same thing: establishing a secure connection between apps hosted on a customer's private network and Sauce Labs cloud virtual machines and real devices. The main differences are:
+Sauce Connect Proxy and Sauce IPSec Proxy accomplish the same thing: establish a secure connection between apps hosted on a customer's private network and Sauce Labs cloud virtual machines and real devices. The main differences are:
```

Comment:
```suggestion
Sauce Connect Proxy and Sauce IPSec Proxy allow you to establish a secure connection between apps hosted on a private network and Sauce Labs cloud virtual machines and real devices. The main differences are:
```

Reply:
Although out of scope for this PR, the descriptions in the table below refer to the reader as "customer", which we should change to "you/your".